packages/rum/src/viewCollection.ts

```diff
@@ -175,3 +198,18 @@ function trackTimings(lifeCycle: LifeCycle, callback: (timings: Timings) => void
   )
   return { stop: stopPerformanceTracking }
 }
+
+export function trackLoadDuration(lifeCycle: LifeCycle, callback: (loadDurationValue: number) => void) {
+  // Stop the current user action
+  stopCurrentUserAction()
```

Comment:
NIT: This has little to do with `loadDuration`, I think it is safe to move in `newView` directly

Reply:
moved at the top of the `function newView(` which makes more sense